coreos-base/oem-packet/files/phone-home.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+curl -s -o /tmp/jq http://images.packet.net/jq
```

Comment:
Posted https://github.com/coreos/coreos-overlay/pull/1291

The alternative would be to ship the jq binary in the OEM partition instead of the base image but the base image should be fine.